src/services/perimeters.ts

```diff
@@ -23,13 +23,24 @@ import fakeFacetClassSimple from '../data/fakeData/facet-class-simple'
 import fakeFacetStartDateFacet from '../data/fakeData/facet-start-date-facet'
 import fakePatients from '../data/fakeData/patients'
 import { FHIR_API_Response, CohortData, ScopeTreeRow } from 'types'
+import { getServicePatientsCount } from './scopeService'
 
 export const getOrganizations = async (ids?: string): Promise<IOrganization[]> => {
   const orgaIdsParam = ids ? `?_id=${ids}` : ''
   const respOrganizations = await api.get<FHIR_API_Response<IOrganization>>(`/Organization${orgaIdsParam}`)
   return getApiResponseResources(respOrganizations) ?? []
 }
 
+export const getPractitionerPerimeter = async (practitionerId: string) => {
+  const resp = await api.get<FHIR_API_Response<IOrganization>>(
+    `/Organization?_has:PractitionerRole:organization:practitioner=${practitionerId}:active=true`
+  )
+  const organizations = getApiResponseResources(resp)
```

Comment:
nit: return [] here to avoid checking for empty stuff in the rest of the function